src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs

```diff
@@ -177,10 +177,6 @@ internal BluetoothLeDevice(BluetoothLeScanData scanData)
         /// </summary>
         ~BluetoothLeDevice()
```

Comment:
Remove empty destructor

Reply:
Done.

Reply:
How about remaining empty function? in order to remove this, ACR required. but, this function can be used in later.